### PR TITLE
implemented cross validation

### PR DESCRIPTION
**Pull request by ryanhoque**

### CODE REVIEW

**Comment by DanielTakeshi** [comment]
Thanks!! Reviewing now

**Comment by DanielTakeshi** [comment]
@ryanhoque Any chance you can add some sample results in the log? Maybe add to the SAMPLE_RESULTS.md readme that I have here, along with the exact command that you used? (Feel free to push any of these directly to master, since it wouldn't be a code change but just changing a .md file, for other code changes pull requests are greatly appreciated.)

I realized, we don't really need train and valid directories. All we need are 10 different files that each refer to their own episodes. Then in training we just pick and choose the one episode we had as validation. Then for the other 9 we concatenate their items.

**Comment by ryanhoque** [comment]
@DanielTakeshi I added to SAMPLE_RESULTS.md and removed the train/valid directories as you suggested

File: src/il_ros_hsr/nets/options.py [hunk 1]
```diff
@@ -19,8 +19,8 @@
 # ------------------------------------------------------------------------------
 
 # For `prepare_data.py`
-RAW_DATA_SSL = 'ssldata2'
-NEW_DATA_SSL = 'ssldata2_pytorch'
+RAW_DATA_SSL = 'ssldata'
+NEW_DATA_SSL = 'ssldata_pytorch'
```
Comment (DanielTakeshi) [inline src/il_ros_hsr/nets/options.py:24]: @ryanhoque I assume this refers to only the newer data, we'll just ignore the first one and treat your latest one as ssldata?

**Comment by DanielTakeshi** [comment]
Thanks for the sample results. They make sense. Iterate 10 times, with 20 held-out samples each time (and 180 training samples). Well, modulo a single bad case there but no big deal (really 179-20 or 180-19 splits).

OK at this point, time to get to plotting and visualizing this stuff. Great job so far.

File: src/il_ros_hsr/nets/options.py (hunk 1 above)
Comment (ryanhoque) [reply src/il_ros_hsr/nets/options.py:24]: Yup that was the intention

**Pull request merged**